reference/customers.v3.yml

```diff
@@ -338,8 +342,8 @@ paths:
 
 
         **Notes**
-
-        Attributes Values can not be updated using Update a Customer. Use the Update a [Customer Attribute Values](https://developer.bigcommerce.com/api-reference/customer-subscribers/v3-customers-api/customer-attribute-values/customersattributevaluesput) endpoint.
+        * Attributes Values can not be updated using Update a Customer. Use the Update a [Customer Attribute Values](https://developer.bigcommerce.com/api-reference/customer-subscribers/v3-customers-api/customer-attribute-values/customersattributevaluesput) endpoint.
+        * channel_ids -- Updating this list of channels a customer can access to may create some side effects in a multi-storefront situation as this list is used to determine which customer account we will use to authenticate a shopper given a channel.
```

Comment:
* channel_ids -- Updating the list of channels a customer can access may create some side effects in a multi-storefront situation. This list determines which customer account we will use to authenticate a shopper given a channel.